File: MAY20/COVID19.cpp
#include <iostream>
using namespace std;

int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int n;
        cin >> n;
        int x[n];
        cin >> x[n];
        int min = n;
        int max = 1;
        int i = 0, c = 1;
        while (i < n - 1){
            if (x[i + 1] - x[i] <= 2){
                c++;
            }
            else{
                if (c > max){
                    max = c;
                }
                if (c < min){
                    min = c;
                }
                c = 1;
            }
            i++;
        }
        if (c > max){
            max = c;
        }
        if (c < min){
            min = c;
        }
        cout << min <<" " + t << max <<" "<< x <<endl;
    }
}

File: MAY20/COVID19.py
t = int(input())
for _ in range(t):
    n = int(input())
    x = list(map(int,input().split()))
    min = n
    max = 0
    i = 0
    c = 1
    while i < n - 1:
        if x[i+1]-x[i]<=2:
            c += 1
        else:
            if c>max:
                max = c
            if c<min:
                min = c
            c=1
        i += 1
    if c>max:
        max = c
    if c<min:
        min = c
    print(min,max)